viral/ensemble_reactivation.py

```
@@ -1218,13 +1256,272 @@ def multiple_sessions() -> None:
+        x_axis=np.arange(-30, 30) / 30,
+        color="blue",
+        label="reactivation",
+    )
+    # shaded_line_plot(
+    #     response_pre,
+    #     x_axis=np.arange(-30, 30) / 30,
+    #     color="orange",
+    #     label="preactivation",
+    # )
+    plt.xlabel("Time (s) from event onset")
+    1 / 0
```

Ruff (0.14.0)

Found useless expression. Either assign it to a variable or remove it.

(B018)